BuildResidentialHPXML/measure.rb

```diff
@@ -4371,6 +4390,11 @@ def self.add_building(hpxml, args)
     return hpxml.buildings[-1]
   end
 
+  # set site properties, including:
+  # - shielding
+  # - ground/soil
+  # - surroundings
+  # - orientation
```

Comment:
Why does this file not follow the documentation style you are using elsewhere? E.g., describe each method argument/param, describe the return value, etc.